recipes/tardis/meta.yaml

```diff
@@ -0,0 +1,107 @@
+{% set name = "tardis-sn" %}
+{% set version = "2.0.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  git_url: https://github.com/tardis-sn/tardis.git
+  git_branch: master
+
+build:
+  number: 0
+  skip: true  # [not win]
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - setuptools
+    - python=3
+    - numpy=1.15
+    - scipy=1.1
+    - pandas=0.24
+    - astropy=3
+    - numexpr
+    - Cython=0.29
+    - networkx
+    - pyyaml
+    - jsonschema
+    - pyne=0.5
+    - pytables
+    - h5py
+    - jupyter
+    - notebook
+    - matplotlib
+    - graphviz
+    - pygraphviz
+    - sphinx
+    - nbconvert
+    - numpydoc
+    - docutils
+    - nbformat
+    - nbsphinx
+    - sphinx_bootstrap_theme
+    - sphinxcontrib-bibtex
+    - sphinx_rtd_theme
+    - git-lfs
+    - pytest
+    - pytest-html
+    - requests
+    - coverage
+    - docopt
+    - pytest-cov
+    - codecov
+    - backports
+
+  run:
+    - python=3
+    - numpy=1.15
+    - scipy=1.1
+    - pandas=0.24
+    - astropy=3
+    - numexpr
+    - Cython=0.29
+    - networkx
+    - pyyaml
+    - jsonschema
+    - pyne=0.5
+    - pytables
+    - h5py
+    - jupyter
+    - notebook
+    - matplotlib
+    - graphviz
+    - pygraphviz
+    - sphinx
+    - nbconvert
+    - numpydoc
+    - docutils
+    - nbformat
+    - nbsphinx
+    - sphinx_bootstrap_theme
+    - sphinxcontrib-bibtex
+    - sphinx_rtd_theme
+    - git-lfs
+    - pytest
+    - pytest-html
+    - requests
+    - coverage
+    - docopt
+    - pytest-cov
+    - codecov
+    - backports
+
+test:
+  imports:
+    - tardis
+
+about:
+  license: BSD-3
+  license_file: LICENSE.rst
+  summary: "TARDIS - Temperature And Radiative Diffusion In Supernovae"
+  home: https://github.com/tardis-sn/tardis
```

Comment:
```suggestion
  home: https://github.com/tardis-sn/tardis
  dev_url: https://tardis.readthedocs.io/en/latest/
  doc_url: https://github.com/tardis-sn/tardis
```

Reply:
dev_url and doc_url are inverted?

Reply:
Yes sorry about that 😆

Reply:
Haha, no problem. It's my first conda-forge package so I was not sure.